src/pages/About/constants.ts

```diff
@@ -15,13 +17,15 @@ export const CARDS = [
     description: 'Buy, sell, and explore tokens on Ethereum, Polygon, Optimism, and more.',
     darkBackgroundImgSrc: darkSwapCardImgSrc,
     lightBackgroundImgSrc: lightSwapCardImgSrc,
+    elementName: ElementName.ABOUT_PAGE_NFTS_CARD,
   },
   {
     to: '/nfts',
     title: 'Trade NFTs',
     description: 'Buy and sell NFTs across marketplaces to find more listings at better prices.',
     darkBackgroundImgSrc: darkNftCardImgSrc,
     lightBackgroundImgSrc: lightNftCardImgSrc,
+    elementName: ElementName.ABOUT_PAGE_SWAP_CARD,
```

Comment:
you need to switch these... NFTs is using elementName for swap right now, and vice versa